### PR TITLE
Visual rules flighting

### PR DESCRIPTION
**Pull request by pasanchMSFT**
### Summary of changes 🔍 
Initial pull request for refactor of Status and Alerts UI. This pull request includes a temporary lightweight flighting solution to be able to render Visual rules in the left panel. This pull request does not contain all scaffolding and type definition, that is still a work in progress.

### Testing 🧪
- Add `cardboard.feature.visualRulesPivot` key and set to true in localStorage.
- Go to Behavior form and see the pivot change.

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRules.types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { IExpressionRangeVisual } from '../../../../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+
+export interface IVisualRulesListProps {
+    ruleList: IExpressionRangeVisual[];
```
Comment (msnyder-msft) [inline src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRules.types.ts:4]: nit: maybe consider `ruleItems`?

File: src/Models/Constants/Constants.ts [hunk 2]
```diff
@@ -82,6 +82,9 @@ export const LOCAL_STORAGE_KEYS = {
     FeatureFlags: {
         Telemetry: {
             debugLogging: 'cardboard.debug.telemetryLogging' // enables debug logging for all emitted telemetry events
+        },
+        VisualRules: {
+            showVisualRulesPivot: 'cardboard.show.visualRulesPivot' // shows visual rules pivot
```
Comment (msnyder-msft) [inline src/Models/Constants/Constants.ts:87]: maybe `cardboard.feature.visualRules...`? Just to differentiate debug items from features/

**Review by msnyder-msft — APPROVED** [review]

File: src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/VisualRulesTab.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { VisualRulesList } from '../VisualRules/VisualRuleList';
+
+export const VisualRulesTab: React.FC = () => {
```
Comment (msnyder-msft) [inline src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/VisualRulesTab.tsx:4]: aren't there some common things all the tabs have at the root when it comes to styles or something?

Comment (pasanchMSFT) [reply src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/VisualRulesTab.tsx:4]: yeah, left panel styles, but I don't have any elements to attach styles to atm.

**Review by msnyder-msft — APPROVED** [review]

**Review by abalogun-dev — APPROVED** [review]

**Pull request merged**